src/components/Contacts/ContactDetails/ContactDetailsHeader/ContactDetailsHeader.graphql

```diff
@@ -15,6 +15,38 @@ fragment ContactDetailsHeader on Contact {
   ...ContactHeaderNewsletter
 }
 
+fragment ContactDetails on Contact {
```

Comment:
Can you add the missing `id` field to `primaryPerson` on line 5 while you're in this file?